src/seedu/addressbook/data/person/Address.java

```diff
@@ -36,6 +39,13 @@ public static boolean isValidAddress(String test) {
         return test.matches(ADDRESS_VALIDATION_REGEX);
     }
 
+    /**
+     * Retrieves a listing of every keyword in the address, in order.
+     */
+    public List<String> getKeywordsInAddress() {
+        return Arrays.asList(value.split("[^a-zA-Z\\d]"));
+    }
```

Comment:
For consistency with `getTagValues` perhaps this function should be in `FindCommand.java` file, or vice versa?